src/routes/write-exercise/components/__tests__/AnswerReview.spec.tsx

```diff
@@ -18,27 +18,31 @@ describe('Feedback', () => {
     word: 'Abrissbirne',
   }
 
-  const renderFeedback = (
+  const renderAnswerReview = (
     result: SimpleResult,
     numberOfTries: number,
     submission: string,
     needsToBeRepeated: boolean
   ): RenderAPI => {
     const documentWithResult: DocumentResult = { document, result, numberOfTries }
     return render(
-      <Feedback documentWithResult={documentWithResult} submission={submission} needsToBeRepeated={needsToBeRepeated} />
+      <AnswerReview
+        documentWithResult={documentWithResult}
+        submission={submission}
+        needsToBeRepeated={needsToBeRepeated}
+      />
     )
   }
 
   it('should render correct feedback', () => {
```

Comment:
please also rename the names of the tests

```suggestion
  it('should render correct answer review', () => {
```

Reply:
done